Modules/_hashopenssl.c

```diff
@@ -27,12 +27,13 @@
 #include "pycore_hashtable.h"
 #include "pycore_pyhash.h"        // _Py_HashBytes()
 #include "pycore_strhex.h"        // _Py_strhex()
+#include "pycore_pyatomic_ft_wrappers.h" // FT_ATOMIC_LOAD_PTR_RELAXED
```

Comment:
Can you also backport the `//` alignment of the `strhex.h` and above? I guess it could ease conflicts (since you also aligned the FIPS mode comment)